docs/HOWTO-ar.md

```diff
@@ -1,19 +1,23 @@
 *[إقرأ هذا بلغات أخرى](../README.md#translations)*
 
+<div dir="rtl" markdown="1">
+
 مرحبا بكم في *!Free-Programming-Books* نرحّب بجميع المساهمين الجدد؛ ونرحب أيضا بهؤلاء الذين يريدون تقديم أول بولّ ريكويست لهم علي جيت هبّ. إن كنت واحدا منهم، فإليك بعض المصادر التي ربما تساعدك:
 
-* [ 🇺🇸 عن البولّ ريكويست](https://help.github.com/articles/about-pull-requests/)
-* [ 🇺🇸 إنشاء بولّ ريكويست](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
-* [ 🇺🇸 جيت هبّ مرحبا يا عالَم](https://guides.github.com/activities/hello-world/)
-* [ 🇺🇸 يوتيوب - دورات تعليمية عن جيت هبّ للمبتدئين](https://www.youtube.com/watch?v=0fKg7e37bQE)
-* [ 🇺🇸 يوتيوب - كيف تنشئ نسختك من مستودع علي جيت هبّ وتقوم بتقديم بولّ ريكويست](https://www.youtube.com/watch?v=G1I3HF4YWEw)
-* [ 🇺🇸 يوتيوب - دورة تعليمية مكثفة عن لغة المارك داون](https://www.youtube.com/watch?v=HUBNt18RFbo)
-* [ 🇩🇿 يوتيوب - دورة تعليمية مكثفة عن لغة المارك داون](https://www.youtube.com/watch?v=1lZCkU5VpIs)
-* [ 🇪🇬 يوتيوب - دورات تعليمية عن جيت هبّ للمبتدئين](https://www.youtube.com/playlist?list=PLDoPjvoNmBAw4eOj58MZPakHjaO3frVMF)
+* [عن البولّ ريكويست](https://help.github.com/articles/about-pull-requests/)
```

Comment:
```suggestion
* [عن البولّ ريكويست](https://help.github.com/articles/about-pull-requests/) *(in english)*
```

As Eric comments in https://github.com/EbookFoundation/free-programming-books/pull/6701#issuecomment-1032737122, it's better remove flags and add a note about lang at the end.

Aply in the other items